Login.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { StyleSheet, Button, View, SafeAreaView, Text, Alert, TouchableOpacity, ImageBackground } from 'react-native';
```

Comment:
Oh I guess that answered my last question, does this limit what you're able to do at all?

Reply:
There is a limit to the preset templaets - but there are several templates to choose from